app/screens/user_profile/user_info.tsx

```diff
@@ -17,36 +19,47 @@ type Props = {
     showNickname: boolean;
     showPosition: boolean;
     user: UserModel;
+    enableCustomAttributes?: boolean;
 }
 
-const UserInfo = ({localTime, showCustomStatus, showLocalTime, showNickname, showPosition, user}: Props) => {
-    const {formatMessage} = useIntl();
+const UserInfo = ({localTime, showCustomStatus, showLocalTime, showNickname, showPosition, user, enableCustomAttributes}: Props) => {
     const customStatus = getUserCustomStatus(user);
+    const serverUrl = useServerUrl();
+    const [customAttributes, setCustomAttributes] = useState<DisplayCustomAttribute[]>([]);
+
+    useEffect(() => {
+        if (enableCustomAttributes) {
+            const fetchData = async () => {
+                try {
+                    const client = NetworkManager.getClient(serverUrl);
+                    const fields = await client.getCustomProfileAttributeFields();
+                    const attrValues = await client.getCustomProfileAttributeValues(user.id);
+                    if (fields) {
+                        const attributes = fields.map((field) => ({
+                            id: field.id,
+                            name: field.name,
+                            value: attrValues[field.id] || '',
```

Comment:
How "controlled" are the field ids? If they are completely user provided, consider using here `secureGetFromRecord` to avoid security issues.